addons/event/static/src/client_action/event_registration_summary_dialog.xml

```diff
@@ -72,8 +82,7 @@
                 </div>
             </div>
             <t t-set-slot="footer">
-                <button t-if="needManualConfirmation" class="btn btn-primary" t-on-click="() => this.onRegistrationConfirm()">Confirm</button>
-                <button t-ref="continueButton" class="btn btn-primary" t-on-click="() => this.onScanNext()">Continue</button>
+                <button class="btn btn-primary" t-on-click="() => this.onRegistrationConfirm()">Continue</button>
```

Comment:
Should we then add the t-ref on the new Continue? Since otherwise the JS is refering to nothing?

Reply:
Or actually is it even used anymore? Do we keep the feature allowing to scan in a row?

Reply:
Yes,  if scanned from camera and when user clicks on continue, then it'll open the camera again (`onScanNext()`) to scan in a row.